apps/SageAccountWeb/src/App.tsx

```diff
@@ -51,131 +52,143 @@ function App() {
         <StyledEngineProvider injectFirst>
           <QueryClientProvider client={queryClient}>
             <ApplicationSessionManager>
-              <AppInitializer>
-                <CookiesNotification />
-                <Switch>
-                  <Route exact path="/">
+              <SourceAppProvider>
```

Comment:
Diff is bad, the only change here is that I added this `SourceAppProvider` to wrap `AppInitializer`